### PR TITLE
metadata related updates to unit tests and batch jobs

### PR DESCRIPTION
**Pull request by solomon-negusse**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Make sure you are requesting to pull a topic/feature/bugfix branch (right side). Don't request your master!
- [ ] Make sure you are making a pull request against the develop branch (left side). Also you should start your branch off our develop.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Unit tests  and batch jobs would fail with the metadata schema changes 

## What is the new behavior?

- Existing and new metadata unit tests would pass
- Asset creation batch jobs will succeed 

## Does this introduce a breaking change?

- [x] Yes
- [ ] No

Migration details to follow - will link to playbook before deploying this.

## Other information

This is a nominal PR that breaks up the large [metadata](https://github.com/wri/gfw-data-api/pull/279) PR to make it easier to review but won't be merged/deployed.

### CODE REVIEW

File: app/tasks/static_vector_tile_cache_assets.py [hunk 1]
```diff
@@ -197,8 +196,8 @@ async def _get_vector_tile_server(
 
     response = {
         "currentVersion": 10.7,
-        "name": metadata["title"],
-        "copyrightText": metadata["citation"],
+        "name": version,
```
Comment (dmannarino) [inline app/tasks/static_vector_tile_cache_assets.py:199]: Is this correct? Looks to me that the version var holds the dataset version (as you'd expect). Is that really what's expected in the "name" field?

File: tests/crud/test_assets.py [hunk 2]
```diff
@@ -203,42 +194,51 @@ async def test_assets_metadata():
...
-        "source": "Source",
-        "version_number": version,
-    }
-
-    async with ContextEngine("WRITE"):
-        asset = await update_asset(asset_id, metadata={"source": "Source"})
-    assert asset.metadata == result_metadata
+    # async with ContextEngine("WRITE"):
+    #     asset = await update_asset(asset_id, metadata={"source": "Source"})
+    # assert asset.metadata == result_metadata
```
Comment (dmannarino) [inline tests/crud/test_assets.py:237]: Is there a reason to keep this around?

**Review by dmannarino — APPROVED** [review]

File: app/tasks/static_vector_tile_cache_assets.py (hunk 1 above)
Comment (solomon-negusse) [reply app/tasks/static_vector_tile_cache_assets.py]: Good catch.  This is fixed.

File: tests/crud/test_assets.py (hunk 2 above)
Comment (solomon-negusse) [reply tests/crud/test_assets.py]: Updated the test. Thanks!

**Pull request closed without merging**